files/build_templates/gbsyncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=gbsyncd service
+Requires=database.service updategraph.service
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules.service
+ConditionPathExists=!/usr/share/sonic/hwsku/gearbox_config.json
```

Comment:
i think gearbox_config should not only be available for broadcom platform.

Reply:
Done